azkaban-web-server/src/main/java/cloudflow/services/ExecutionServiceImpl.java

```diff
@@ -264,8 +266,47 @@ private void setAzkabanExecutionOptions(ExecutionParameters executionParameters,
     executionOptions.setNotifyOnLastFailure(executionParameters.isNotifyFailureOnExecutionComplete());
 
     executionOptions.setConcurrentOption(executionParameters.getConcurrentOption().getName());
+    setRuntimeProperties(executionParameters, executionOptions);
 
     executableFlow.setExecutionOptions(executionOptions);
   }
 
+  private void setRuntimeProperties(ExecutionParameters executionParameters,
+      ExecutionOptions executionOptions) {
+
+    if (executionParameters.getProperties().size() == 0) {
+      return;
+    }
+    // supporting only flow level properties overwrite for the POC
+    if(executionParameters.getProperties().size() > 1 ||
+        !executionParameters.getProperties().containsKey("root")) {
+      logger.error("Attempt to overwrite properties of nodes other than the root flow detected: "
+          + executionParameters.getProperties());
+      throw new CloudFlowException("Overwriting properties of jobs or nested flows is not "
+          + "yet supported.");
+    }
+
+    Map<String, String> runtimeProps = executionParameters.getProperties().get("root");
+    if(runtimeProps.containsKey(CommonJobProperties.FAILURE_EMAILS)) {
+      String rawEmails = runtimeProps.get(CommonJobProperties.FAILURE_EMAILS);
+      if (!rawEmails.isEmpty()) {
+        String[] emails = rawEmails.split("\\s*,\\s*|\\s*;\\s*|\\s+");
+        executionOptions.setFailureEmails(Arrays.asList(emails));
+        executionOptions.setFailureEmailsOverridden(true);
+      }
+      runtimeProps.remove(CommonJobProperties.FAILURE_EMAILS);
+    }
+
+    if(runtimeProps.containsKey(CommonJobProperties.SUCCESS_EMAILS)) {
```

Comment:
Same comments as above.

Reply:
To maintain compatibility with current Azkaban code. They should not be removed in CF.